Content.Shared/Mobs/Systems/MobStateSystem.StateMachine.cs

```diff
@@ -36,7 +36,8 @@ public void UpdateMobState(EntityUid entity, MobStateComponent? component = null
     }
 
     /// <summary>
-    /// Change the MobState and trigger MobState update events
+    /// Change the MobState without triggering UpdateMobState events.
+    /// WARNING: use this sparingly when you need to override other systems (MobThresholds)
```

Comment:
this is rly sus to just completely bypass the state machine, is there not another way you can do this

Reply:
I talked with jezi about it and we ultimately concluded it was best to just have the functionality but keep it limited through review. Otherwise, the state machine becomes authoritative to the point of hampering a lot of interactions like revives, etc.